File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.java
class Solution {
    public int solution(int[] arr1, int[] arr2) {
        int answer = 0;
        if(arr1.length == arr2.length){
            int temp1 = 0;
            int temp2 = 0;
            for(int i = 0; i< arr1.length; i++){
                temp1 += arr1[i];
                temp2 += arr2[i];
            }
            if(temp1 > temp2){
                answer = 1;
            }else if(temp1 < temp2){
                answer = -1;
            }else {
                answer = 0;
            }
        }else if(arr1.length < arr2.length){
            answer = -1;
        }else{
            answer = 1;
        }
        return answer;
    }
}

File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.js
function solution(arr1, arr2) {
    var answer = 0;
    var compareSum1 = 0;
    var compareSum2 = 0;
    arr1.map(item => compareSum1 += item)
    arr2.map(item => compareSum2 += item)
    arr1.length > arr2.length ? answer = 1 : answer = -1
    if( arr1.length === arr2.length){
        if(compareSum1 > compareSum2){
            answer = 1
        } else if( compareSum1 < compareSum2 ){
            answer = -1
        } else {
            answer=0
        }
    }
    return answer
}